src/dbnode/namespace/convert.go

```diff
@@ -80,6 +82,25 @@ func ToIndexOptions(
 	return iopts, nil
 }
 
+// ToRuntimeOptions converts nsproto.NamespaceRuntimeOptions to RuntimeOptions.
+func ToRuntimeOptions(
+	opts *nsproto.NamespaceRuntimeOptions,
+) (RuntimeOptions, error) {
```

Comment:
nit: Doesn't look like this can error, or is this just future proofing?

Reply:
This is for future proofing yeah to make sure people don't rely on the API and then make it very difficult to propagate errors from a code path that never had to later.